yt_dlp/extractor/youtube.py

```diff
@@ -3846,8 +3902,12 @@ def append_client(*client_names):
                 experiments = traverse_obj(pr, (
                     'responseContext', 'serviceTrackingParams', lambda _, v: v['service'] == 'GFEEDBACK',
                     'params', lambda _, v: v['key'] == 'e', 'value', {lambda x: x.split(',')}, ...))
-                if all(x in experiments for x in self._POTOKEN_EXPERIMENTS):
+                if all(x in experiments for x in self._POTOKEN_EXPERIMENTS) and not self._get_cached_po_token(client=client):
```

Comment:
Add a check that will only enter this branch if the client is not already marked as requires PO Token